docs/sources/mimir/configure/configuration-parameters/index.md

```diff
@@ -3635,6 +3635,13 @@ The `limits` block configures default and per-tenant limits imposed by component
 # CLI flag: -store.max-labels-query-length
 [max_labels_query_length: <duration> | default = 0s]
 
+# Maximum number of items, series queries. This limit is enforced in the
```

Comment:
I'm a little confused by the phrasing here. Is this the maximum number of series queries permitted? If so, we can remove the `items` and just say `Maximum number of series queries`.